applications/ocelloids_xcm_monitoring_service.md

```diff
@@ -0,0 +1,270 @@
+# Ocelloids XCM Transfer Monitoring Service
+
+- **Team Name:** SO/DA zone
+- **Payment Address:** Fiat 04.05.2023, 16:37
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Project Overview
+
+This grant proposal is a follow-up to the Ocelloids Monitoring SDK, previously delivered and available here: https://github.com/w3f/Grant-Milestone-Delivery/pull/934
+
+The objective of this grant is to develop an open-source monitoring service using the Ocelloids SDK. This service will monitor XCM transfers across Asset Hub and selected parachains. The primary purpose is to offer exchanges, custodians and other institutions on Asset Hub an efficient means of tracking asset transfers across different parachains. Additionally, we will abstract the underlying pallets responsible for initiating XCM transfers and managing assets, which varies for each parachain. Users will have access to an HTTP API to subscribe to XCM transfers and manage their subscriptions. A Docker image will be provided to facilitate service deployment.
```

Comment:
1. We have updated the description to make it more clear.
2. After reviewing the monitoring flow we modified it to avoid dependencies on specific pallet implementations. No, we won't provide an API to transact.
3. Yes, we are not planning to host the service ourselves.